docs/index.rst

```diff
@@ -59,6 +59,7 @@ User Documentation
    install
    getting_started
    Example gallery <generated/examples/index>
+   Tutorials <http://tutorials.astropy.org/>
```

Comment:
It's working like this, your suggestion is needed when you want to refer to the link as `Tutorial`_ later.